src/index.html

```diff
@@ -7,7 +7,56 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  
   <body>
-    <h1>Stars</h1>
+    <div class="all-stars">
```

Comment:
![image](https://user-images.githubusercontent.com/103033758/233038063-bf2482c5-3a76-40a1-a212-13465bbce8fd.png)

If that's not what you mean - it is better to rename this class for sth like stars__container

Reply:
Got it!